### PR TITLE
Added new eicons for Elementor 3.1.0, including replacing responsive … (ED-1659)

### PR DESCRIPTION
**Pull request by udidol**
…device icons - Updated E-Icons to v5.10.0.

### CODE REVIEW

File: css/elementor-icons.css [hunk 1]
```diff
@@ -1173,6 +1173,45 @@
 .eicon-background:before {
   content: '\e97d'; }
 
+.eicon-parts:before {
+  content: '\e97f'; }
+
+.eicon-plugins:before {
+  content: '\e980'; }
+
+.eicon-pro:before {
```
Comment (kobizz) [inline css/elementor-icons.css:1182]: We already have `.eicon-pro-icon` that looks the same

File: scss/_icons.scss [hunk 2]
```diff
@@ -380,6 +380,19 @@
...
+.eicon-responsive:before { content: '\e982'; }
+.eicon-upload-1:before { content: '\e983'; }
+.eicon-video-playlist:before { content: '\e985'; }
+.eicon-wide-screen:before { content: '\e986'; }
+.eicon-code-highlighter:before { content: '\e987'; }
+.eicon-default_desktop:before { content: '\e988'; }
+.eicon-details:before { content: '\e98a'; }
+.eicon-dev:before { content: '\e98b'; }
+.eicon-export:before { content: '\e98c'; }
+.eicon-import:before { content: '\e98d'; }
```
Comment (kobizz) [inline scss/_icons.scss:396]: Make sure all icons code are consecutive

**Review by kobizz — CHANGES_REQUESTED** [review]
Make sure that `.eot` and `.ttf` files are included

File: scss/_icons.scss [hunk 3]
```diff
@@ -380,6 +380,17 @@
 .eicon-globe:before { content: '\e97b'; }
 .eicon-typography-1:before { content: '\e97c'; }
 .eicon-background:before { content: '\e97d'; }
+.eicon-device-base:before { content: '\e97e'; }
```
Comment (kobizz) [inline scss/_icons.scss:383]: This icon has to be removed

File: index.html [hunk 4]
```diff
@@ -264,7 +264,7 @@
         /* remove if not needed */
         margin-left: .2em;
 
-        /* You can be more comfortable with increased icons size */
+        /* You can be more comfortable with increased eicons size */
```
Comment (kobizz) [inline index.html:267]: ?

**Review by kobizz — CHANGES_REQUESTED** [review]
What happened to `.eot` and `.ttf` files?

File: css/elementor-icons.css [hunk 5]
```diff
@@ -1173,6 +1173,45 @@
...
+.eicon-video-playlist:before {
+  content: '\e985'; }
+
+.eicon-wide-screen:before {
+  content: '\e986'; }
+
+.eicon-code-highlighter:before {
+  content: '\e987'; }
+
+.eicon-default_desktop:before {
```
[the remark this answers is not in the document]
Comment (udidol) [reply css/elementor-icons.css]: removed.

File: scss/_icons.scss (hunk 3 above)
Comment (udidol) [reply scss/_icons.scss]: Done.

File: index.html (hunk 4 above)
Comment (udidol) [reply index.html]: Fixed.

**Review by kobizz — APPROVED** [review]

**Pull request merged**